packages/cubejs-snowflake-driver/driver/SnowflakeDriver.js

```diff
@@ -74,6 +74,9 @@ class SnowflakeDriver extends BaseDriver {
       database: process.env.CUBEJS_DB_NAME,
       username: process.env.CUBEJS_DB_USER,
       password: process.env.CUBEJS_DB_PASS,
+      authenticator: process.env.CUBEJS_DB_AUTHENTICATOR,
```

Comment:
We need to use the `SNOWFLAKE_` prefix because this will indicate that these variables work only with SnowFlake.

Reply:
Done ✅